python/lsst/meas/algorithms/convertReferenceCatalog.py

```diff
@@ -64,6 +66,157 @@ def addRefCatMetadata(catalog):
     catalog.setMetadata(md)
 
 
+def _makeSchema(filterNameList, *, addCentroid=False,
+                addIsPhotometric=False, addIsResolved=False,
+                addIsVariable=False, fullPositionInformation=False):
+    """Make a standard schema for reference object catalogs.
+
+    Parameters
+    ----------
+    filterNameList : `list` of `str`
+        List of filter names. Used to create <filterName>_flux fields.
+    addCentroid : `bool`
+        If True then add fields "centroid" and "hasCentroid".
+    addIsPhotometric : `bool`
+        If True then add field "photometric".
+    addIsResolved : `bool`
+        If True then add field "resolved".
+    addIsVariable : `bool`
+        If True then add field "variable".
```

Comment:
These parameter names are copied from the original `makeMinimalSchema`. I'm fine with changing them since we're changing so much already, unless @parejkoj knows of some historical reason for having "Is" in the name.

Reply:
Thinking about this a little more, I think keeping the "Is" would be better, since these correspond to adding boolean fields. To me "addVariable", for example, makes me wonder if some scalar parameter is added, whereas "addIsVariable" more clearly implies that the resulting flag with correspond to "Is Variable" or "Is Not Variable".